tapir/coop/templates/coop/membership_agreement_pdf.default.html

```diff
@@ -0,0 +1,259 @@
+{% load static %}
+{% load django_bootstrap5 %}
+{% load accounts %}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Beteiligungserklärung {{ owner.first_name }} {{ owner.last_name }}</title>
+    <link rel="stylesheet" href="{% static "coop/css/membership_agreement_pdf.css" %}">
+</head>
+<body>
+
+<!-- Falzmarken -->
+<span style="border-top: 1px solid grey; width: 5mm; position: fixed; left: -15mm; top: 60mm;"></span>
+<span style="border-top: 1px solid grey; width: 5mm; position: fixed; left: -15mm; top: 165mm;"></span>
+
+<!-- A DIN 5008 Letter has 2,7cm of top margin -->
+<div style="display: flex">
+    <div style="margin-top: 17.7mm; margin-bottom: 30mm; margin-right: 30mm;">
+        {% if owner %}
+        <span style="font-size: 6pt;">
+            {{ owner.first_name }} {{ owner.last_name }},
+            {% if owner.street and owner.city %}
+                {% if owner.street_2 %} {{ owner.street_2 }},{% endif %}
+                {{ owner.street }},
+                {{ owner.postcode }} {{ owner.city }}
+            {% endif %}
+        </span>
+        {% endif %}
+        <div style="margin-top: 2mm;">
+            Organisation<br />
+            Straße 1<br />
+            12345 Stadt
+        </div>
+    </div>
+    <div style="flex-grow: 1"></div>
+    <div style="margin-top: 13mm;">
+        <img src="{% static "../static/images/Logo_dark.svg" %}" style="width: 60mm;" />
+    </div>
+</div>
+
+<div style="text-align: center; font-size: 14pt; font-weight: bold;">
+    Beteiligungserklärung <em>Organisation</em>
```

Comment:
Should we put COOP_NAME here? Or do we not expect anybody to use this anyway and instead keep it as a template?

If we expect the default template to not be used in any production setting anyway, let's put a comment in the rendering code and the template to indicate that this template is really only expected to be rendered in non-customized "fresh" versions of the code and should always be adapted per Coop.

Reply:
Yes, I've but in placeholders to make people completely adapt the whole template.
I'll add the remarks!